tests/spec/fs.symlink.spec.js

```diff
@@ -43,4 +43,81 @@ describe('fs.symlink', function() {
       });
     });
   });
+
+  // it('should be return an error if the file destination path is bot exists', function(done){
+  //   var fsPromises = unil.fs().promises;
+
+  //   fsPromises.symlink('/tmp', '/', function(error){
+  //     expect(error).to.exist;
+  //     expect(error.code).to.equal('EEXIST');
+  //     done();
+  //   });
+  // });
 });
+
+describe('fs.promises.symlink', function(){
+  beforeEach(function(done){
+    util.setup(function(){
+      var fs = util.fs();
+      fs.promises.symlink('/','myfile')
+        .then(done())
+        .catch(function(error){
+          throw error;
+        });
+    });
+  });
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.symlink).to.be.a('function');
+  });
+
+  // it('should be return an error if the file destination path is bot exists', function(done){
```

Comment:
I think you should uncomment this part since you want to test the case when the path does not exist.

Reply:
Okay. I do this right now.